src/components/screens/SchoolScheduleCreation.js

```diff
@@ -55,38 +53,48 @@ class SchoolScheduleCreation extends React.Component {
 						else this.props.navigation.pop();
 					});
 				})
-				.catch(err => console.log('eerr', err));
+				.catch(err => console.log('error', err));
 		}
 	}
 
-	progressValue() {
-		return setInterval(() => {
-			if (this.state.width <= 1) {
-				let width  = this.state.width + 0.01;
-				this.setState({width});
-			}
-		},10);
-	}
-
 	render() {
 		return(
 			<LinearGradient style={styles.container} colors={gradientColors}>
-				<ImageBackground style={styles.container} source={require('../../assets/img/loginScreen/backPattern.png')} resizeMode="repeat">
-					<StatusBar translucent={true} backgroundColor={'rgba(0, 0, 0, 0.4)'} />
+				<ImageBackground style={styles.container} 
+					source={require('../../assets/img/loginScreen/backPattern.png')} 
+					resizeMode="repeat"
+				>
+					<StatusBar translucent={true} 
+						backgroundColor={'rgba(0, 0, 0, 0.4)'} 
+					/>
 					<Surface style={styles.surface}>
 						<Text style={styles.title}>Analysing your Picture</Text>
 						<Text style={styles.subtitle}>Extracting the information from your picture</Text>
-
-						<Progress.Bar style={{alignSelf:'center'}} indeterminate={true} width={200} color={blueColor} useNativeDriver={true} borderWidth={0} unfilledColor={orangeColor}/>
-						{/* <ProgressBar style={{height: 10}} progress={this.state.width} color={orangeColor} />
-						<ProgressBarAndroid styleAttr="Horizontal" color={orangeColor}></ProgressBarAndroid> */}
+						<Progress.Bar style={{alignSelf:'center'}} 
+							indeterminate={true} 
+							width={200} 
+							color={blue} 
+							useNativeDriver={true} 
+							borderWidth={0} 
+							unfilledColor={orange}
+						/>
```

Comment:
Put the closing bracket on the same line of the last prop (we forgot to add it in the coding convention)